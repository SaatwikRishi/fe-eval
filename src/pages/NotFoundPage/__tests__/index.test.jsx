import { render, screen } from '@testing-library/react'
import NotFoundPage from '..'
import React from 'react'

describe('NotFoundPage', () => {
  it('should render when component is called', () => {
    const { container } = render(<NotFoundPage />)
    // Assert
    expect(container).toMatchSnapshot()
    expect(screen.getByText('Page Not Found')).toBeTruthy()
  })
})
