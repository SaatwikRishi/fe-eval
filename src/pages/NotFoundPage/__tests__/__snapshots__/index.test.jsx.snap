// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render when component is called 1`] = `
<div>
  <div
    class="not-found-page"
  >
    <h1>
      Page Not Found
    </h1>
  </div>
</div>
`;
